docs/plugins/store/oras.md

```diff
@@ -44,8 +44,8 @@ spec:
 
 | Name           | Required | Description                                                                                                                                                                                                        | Default Value               |
 | -------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------- |
-| cosignEnabled  | no       | This must be `true` if cosign verifier is enabled. Read more about cosign verifier [here](https://github.com/deislabs/ratify/blob/main/plugins/verifier/cosign/README.md).                                         | `false`                     |
-| authProvider   | no       | This is only required if pulling from a private repository. For all supported auth mode, please review [oras-auth-provider](https://github.com/deislabs/ratify/blob/main/docs/reference/oras-auth-provider.md) doc | dockerAuth                  |
+| cosignEnabled  | no       | This must be `true` if cosign verifier is enabled. Read more about cosign verifier [here](https://github.com/ratify-project/ratify/blob/main/plugins/verifier/cosign/README.md).                                         | `false`                     |
```

Comment:
Not for this PR probably. We need a deadlink checker , some of these links has moved.